portal-test/src/com/liferay/portal/kernel/test/util/OrganizationTestUtil.java

```diff
@@ -101,6 +101,13 @@ public static Organization addOrganization(boolean site) throws Exception {
 			RandomTestUtil.randomString(), site);
 	}
 
+	public static Organization addOrganization(long parentOrganizationId)
+		throws Exception {
+
+		return addOrganization(
+			parentOrganizationId, RandomTestUtil.randomString(), false);
+	}
+
```

Comment:
Not needed, in the test you can pass these params to the already existing method